apps/smime.c

```diff
@@ -356,6 +356,8 @@ int smime_main(int argc, char **argv)
             break;
         }
     }
+
+    /* Extra arguements are files with recipient keys. */
```

Comment:
NIT: arguments

Reply:
Line 404 should probably have a line similiar to cms so that
````
openssl smime certname
````
gives some more useful information...
No operation option (-encrypt|-decrypt|-sign|-verify|...) specified.

Reply:
moved the ```if (!operation``` and added an error message.  Fixed typo in comment.